packages/components/src/components/ExternalCard/elements.ts

```diff
@@ -0,0 +1,83 @@
+import styled from 'styled-components';
+import { Stack } from '../Stack';
+import { Text } from '../Text';
+
+export const StyledWrapper = styled(Stack)<{ thumbnail: string }>`
+  position: relative;
+  overflow: hidden;
+  height: 156px;
+  width: 276px;
+  background: #999999;
+  border-radius: 4px;
+
+  &:after {
+    position: absolute;
+    content: '';
+    display: block;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    background-image: url(${props => props.thumbnail});
+    background-size: cover;
+    background-position: center;
+    transform: scale(1);
+    transition: transform ease-in-out;
+    transition-duration: 100ms;
+  }
+
+  &:hover::after {
+    transform: scale(1.01);
+  }
+`;
+StyledWrapper.defaultProps = {
+  direction: 'vertical',
+};
+
+export const StyledContent = styled(Stack)`
+  position: absolute;
+  z-index: 1;
+  width: 100%;
+  height: 100%;
+  top: 0;
+  left: 0;
+  display: flex;
+  background: linear-gradient(
+      0deg,
+      rgba(25, 25, 25, 0.3) 0%,
+      rgba(25, 25, 25, 0) 100%
+    ),
+    linear-gradient(0deg, rgba(25, 25, 25, 0.6) 0%, rgba(25, 25, 25, 0) 100%),
+    linear-gradient(0deg, rgba(25, 25, 25, 0.8) 0%, rgba(25, 25, 25, 0) 100%);
+`;
+
+export const StyledDetails = styled(Stack)`
+  margin: auto 24px 24px;
+`;
+StyledDetails.defaultProps = {
```

Comment:
It seems that [`defaultProps` will be deprecated one day](https://github.com/reactjs/rfcs/blob/createlement-rfc/text/0000-create-element-changes.md#deprecate-defaultprops-on-function-components) (or already is in v18, dunno) so I think we should move these to the component implementation.

Reply:
@tristandubbeld oh, I wasn't aware. Removed the default props here: https://github.com/codesandbox/codesandbox-client/pull/7217/commits/24bed18cc0845f5ca5698afbf07a6ef69cecb37e.